src/transactions/transaction/transactionBuilder.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2019 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import crypto from 'crypto';
```

Comment:
Maybe not the PR to fix this, but node has its inbuilt crypto module now. And `crypto` is deprecated, is the any reason it's being used?

Reply:
This is the built in crypto that ships with Node. It still needs to be imported. https://nodejs.org/api/crypto.html

Reply:
unused package removed in PR #14